README.md

```diff
@@ -74,3 +74,23 @@ The default `print` parameter value is `true`. It means that in this example the
 
 ### Print as a function
 The `print` parameter can also be a function. Then the `props` and `context` are given to it (in this order), and it should return a boolean indicating wether or not the actual component should be displayed.
+
+### Error handling
+The `error` parameter allow to specify a prop that indicates wether or not a placeholder error component should be displayed in replacement of the real component.
+It's usefull when data that are mandatory for the correct display of a component are missing.
```

Comment:
I think this is @hgwood, but someone once tells me to use `required` instead of `mandatory`.
I can't remember why tho.

Reply:
It's just way more common to use "required". "mandatory" is more formal. In general shorter words are preferred (eg speed vs velocity, give up vs abandon, build vs construct...). I often use the shortcut that if it's French-sounding, then it's formal :).